tests/test_access_url.py

```diff
@@ -39,14 +39,19 @@ def test_use_access_url(self):
         onetime = db.session.merge(onetime)
 
         token = user_manager.token_manager.generate_token(onetime.id)
-        access_url = url_for('portal.access_via_token', token=token)
+        access_url = url_for(
+            'portal.access_via_token', token=token, cookies_tested=True,
```

Comment:
@pbugni Did you need to add `cookies_tested=True` to work around these tests because they do not use cookies?...

I don't that QS parameter will prevent the redirection, it's added after redirecting through `/require_cookies` but isn't checked otherwise.

Reply:
no - just testing that the redirection to challenge happens.  but thanks for noticing - that param is no longer needed, removing from the tests.